data/json/npcs/factions.json

```diff
@@ -60,7 +61,9 @@
     "known_by_u": false,
     "size": 100,
     "power": 100,
-    "fac_food_supply": { "calories": 115200, "vitamins": { "iron": 800, "calcium": 800, "vitC": 600 } },
+    "//": "90 days worth of food for 10 people, at 3000kcal/day and full RDA of iron/calcium/vitC",
+    "//2": "Expect to see these values a lot.",
+    "fac_food_supply": { "calories": 2700000000, "vitamins": { "iron": 86400, "calcium": 86400, "vitC": 86400 } },
```

Comment:
Is it worth writing down somewhere in the doc? since it will probably be usefull again in the future

Reply:
I don't really think so. This is a baseline calculation, actual kcal needs per-person vary and the vitamins are subject to change.

In addition I'd point out that I'm using 3000kcal to be 'safe' due to the variance. I think our average character BMR falls between 2500-3000 nowadays, but in the past we had assumed 2500kcal/day/person. 

https://github.com/CleverRaven/Cataclysm-DDA/blob/140db31b8fabe28a7714187bbd7fe88766396ebb/src/itype.h#L118